cyberark-secrets-provider-for-kubernetes/templates/_helpers.tpl

```diff
@@ -0,0 +1,18 @@
+{{/*
+Return the most recent RBAC API available
+*/}}
+{{- define "conjur.rbac-api" -}}
+{{- if .Capabilities.APIVersions.Has "rbac.authorization.k8s.io/v1" }}
+{{- printf "rbac.authorization.k8s.io/v1" -}}
+{{- else if .Capabilities.APIVersions.Has "v1" }}
+{{- printf "v1" -}}
+{{- end }}
+{{- end }}
+
+{{- define "conjur.deploy-api" -}}
```

Comment:
Add comment like the one above the RBAC to explain what this func is for.

Reply:
I think the above comment also applies here. Or do you have an idea?
```
{{/*
Return the most recent Deployment Configuration API available
*/}}
```
?